root/documentation/vep.conf

```diff
@@ -128,6 +133,51 @@
         default=0
       </merged>
+      <gencode_basic>
+        type=Boolean(0,1)
+        description=Limit your analysis to transcripts belonging to the GENCODE basic set. This set has fragmented or problematic transcripts removed.
+        default=0
+      </gencode_basic>
+      <pick>
+        type=Boolean
+        description=Pick one line or block of consequence data per variant, including transcript-specific columns. Consequences are chosen according to the criteria described <a target="_blank" href="http://www.ensembl.org/info/docs/tools/vep/script/vep_other.html#pick">here</a>, and the order the criteria are applied may be customised with <b>pick_order</b>. This is the best method to use if you are interested only in one consequence per variant.
+        default=0
+      </pick>
+      <pick_allele>
+        type=Boolean
+        description=Like <b>pick</b>, but chooses one line or block of consequence data per variant allele. Will only differ in behaviour from <b>pick</b> when the input variant has multiple alternate alleles.
+        default=0
+      </pick_allele>
+      <per_gene>
+        type=Boolean
+        description=Output only the most severe consequence per gene. The transcript selected is arbitrary if more than one has the same predicted consequence. Uses the same ranking system as <b>pick</b>.
+        default=0
+      </per_gene>
+      <pick_allele_gene>
+        type=Boolean
+        description=Like <b>pick_allele</b>, but chooses one line or block of consequence data per variant allele and gene combination.
+        default=0
+      </pick_allele_gene>
+      <flag_pick>
+        type=Boolean
+        description=As per <b>pick</b>, but adds the PICK flag to the chosen block of consequence data and retains others.
+        default=0
+      </flag_pick>
+      <flag_pick_allele>
+        type=Boolean
+        description=As per <b>pick_allele</b>, but adds the PICK flag to the chosen block of consequence data and retains others.
+        default=0
+      </flag_pick_allele>
+      <flag_pick_allele_gene>
+        type=Boolean
+        description=As per <b>pick_allele_gene</b>, but adds the PICK flag to the chosen block of consequence data and retains others.
+        default=0
+      </flag_pick_allele_gene>
+      <pick_order>
+        type=String
+        description=Comma-separated list with order of criteria (and the list of criteria) applied when choosing a block of annotation data with one of the following options: <b>pick</b>, <b>pick_allele</b>, <b>per_gene</b>, <b>pick_allele_gene</b>, <b>flag_pick</b>, <b>flag_pick_allele</b>, <b>flag_pick_allele_gene</b>. See <a target="_blank" href="http://www.ensembl.org/info/docs/tools/vep/script/vep_other.html#pick_options">this page</a> for the default order. Valid criteria are: [ canonical appris tsl biotype ccds rank length mane ].
+        default=canonical,appris,tsl,biotype,ccds,rank,length,mane
```

Comment:
The order is being updated to report MANE first for human: https://github.com/Ensembl/ensembl-vep/pull/1314

Reply:
Change reflected in https://github.com/Ensembl/ensembl-rest/pull/571/commits/b3b1dda9cd84af01de6913c0c616bb3b16399445